snap/validate_test.go

```diff
@@ -1439,6 +1439,17 @@ func (s *validateSuite) TestValidateDescription(c *C) {
 	c.Check(ValidateDescription(strings.Repeat("x", 4000)), IsNil)
 }
 
+func (s *validateSuite) TestValidateTitle(c *C) {
+	for _, s := range []string{
+		"xx", // boringest ASCII
+		"🐧🐧", // len("🐧🐧") == 8
+		"á", // á (combining)
```

Comment:
This one counts as two codepoints?

Reply:
Because it is combining. It would be one code point after normalisation.